types/vscode/index.d.ts

```diff
@@ -10497,6 +10915,23 @@ declare module 'vscode' {
         // Properties: see details [here](https://microsoft.github.io/debug-adapter-protocol/specification#Types_Source).
     }
 
+    /**
+     * A DebugConfigurationProviderTriggerKind specifies when the `provideDebugConfigurations` method of a `DebugConfigurationProvider` is triggered.
+     * Currently there are two situations: to provide the initial debug configurations for a newly created launch.json or
+     * to provide dynamically generated debug configurations when the user asks for them through the UI (e.g. via the "Select and Start Debugging" command).
+     * A trigger kind is used when registering a `DebugConfigurationProvider` with #debug.registerDebugConfigurationProvider.
+     */
+    export enum DebugConfigurationProviderTriggerKind {
+        /**
+         *	`DebugConfigurationProvider.provideDebugConfigurations` is called to provide the initial debug configurations for a newly created launch.json.
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/vscode/index.d.ts;
  Line: 10926;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.